service/tests/test_client.rs

```diff
@@ -280,6 +280,9 @@ async fn test_get_metadata(#[case] vrf_difficulty: Option<[u8; 32]>) {
         16,
         post::config::Cores::Any(1),
         post::pow::randomx::RandomXFlag::get_recommended_flags(),
+        None,
+        1,
+        1,
```

Comment:
And you wouldn't need to provide useless values for the case when remote is disabled:
```suggestion
        None,
```